src/test/java/teammates/test/cases/browsertests/AdminHomePageUiTest.java

```diff
@@ -86,12 +86,12 @@ private void testCreateInstructorAction() throws Exception {
         BackDoor.deleteCourse(demoCourseId);
         BackDoor.deleteInstructor(demoCourseId, instructor.email);
         homePage.createInstructorByInstructorDetailsSingleLineForm(instructorDetails);
```

Comment:
The thing is, the retries we're doing are too small (just a few statements, like a page refresh) and heavily dependent on the current state; it's not ideal to extract them into small tests of their own just to retry them.